app/models/anime.rb

```diff
@@ -83,16 +83,16 @@ class Anime < ActiveRecord::Base
   }
 
 
-  def poster_image_thumb
+  def poster_image_thumb(size = :large)
     if self.poster_image_file_name.nil?
       "https://hummingbird.me/assets/missing-anime-cover.jpg"
     else
       # This disgusting fastpath brought to you by the following issue:
       # https://github.com/thoughtbot/paperclip/issues/909
       if Rails.env.production?
-        "https://static.hummingbird.me/anime/poster_images/#{"%03d" % (self.id/1000000 % 1000)}/#{"%03d" % (self.id/1000 % 1000)}/#{"%03d" % (self.id % 1000)}/large/#{URI.escape(self.poster_image_file_name.rpartition('.')[0])}.jpg?#{self.poster_image_updated_at.to_i}"
+        "https://static.hummingbird.me/anime/poster_images/#{"%03d" % (self.id/1000000 % 1000)}/#{"%03d" % (self.id/1000 % 1000)}/#{"%03d" % (self.id % 1000)}/#{size.to_s}/#{URI.escape(self.poster_image_file_name.rpartition('.')[0])}.jpg?#{self.poster_image_updated_at.to_i}"
```

Comment:
This fix won't work in production? The file extension is hardcoded to `jpg` here.